<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_create_room">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuen Raum anlegen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_sim_data">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Raumdaten Anzeigen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exec_sim">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation starten</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="brw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>451</width>
      <height>361</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Simulationswerte ...</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>281</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sim_data">
     <item>
      <widget class="QLabel" name="lbl_room_no">
       <property name="text">
        <string>Raum Nr.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_room_name">
       <property name="text">
        <string>Raumbezeichnung:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_target_temp">
       <property name="text">
        <string>Temperatur (SOLL):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_target_humidity">
       <property name="text">
        <string>Luftfeuchtigkeit (SOLL):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_thermometer_id">
       <property name="text">
        <string>Thermometer-ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_current_temp">
       <property name="text">
        <string>Temperatur (IST):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_hygrometer_id">
       <property name="text">
        <string>Hygrometer-ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_relative_humidity">
       <property name="text">
        <string>rel. Luftfeuchtigkeit (IST):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_absolute_humidity">
       <property name="text">
        <string>abs. Luftfeuchtigkeit (IST):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_time">
       <property name="text">
        <string>Uhrzeit:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_starting_temp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Startwert Raumtemperatur [C]:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_starting_temp">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
